src/frontend/src/views/Data/components/LineageTooltip/index.tsx

```diff
@@ -1,31 +1,26 @@
 import React from "react";
 import { Label, Text, Wrapper } from "./style";
 
-export interface Lineage {
-  last_updated: string;
-  lineage: string;
-  confidence: string;
-  version: string;
-}
-
 interface Props {
   lineage: Lineage;
 }
 
 const DISPLAY_ORDER: Array<keyof Lineage> = [
   "lineage",
-  "confidence",
+  "qc_status",
   "version",
   "last_updated",
+  "scorpio_call",
+  "scorpio_support",
 ];
 
 export const LineageTooltip = ({ lineage }: Props): JSX.Element => {
   return (
     <>
       {DISPLAY_ORDER.map((key) => {
         let value = lineage[key];
-        if (key === "confidence") {
-          value = `${value}%`;
+        if (key === "last_updated") {
+          value = `${value?.slice(0, 10)}`;
```

Comment:
can you add a comment on why you need this slice?

Reply:
Current display carries time stamp that's not useful
<img width="248" alt="image" src="https://user-images.githubusercontent.com/20667188/165402294-23062e41-7886-4bc2-8262-b0d319551616.png">